data/ru/comparison-data.json

```diff
@@ -2600,7 +2600,7 @@
           },
           {
             "FeatureName": "Другим Параметрам и Возможностям",
-            "Telegram": "advantage|[Широкая поддержка](https://t.me/DeepLink)",
+            "Telegram": "advantage|Широкая поддержка: [1](https://t.me/DeepLink), [2](https://telegram.org/blog/infinite-reactions-statuses#new-username-links)",
```

Comment:
It also makes sense to update https://t.me/DeepLink to https://corefork.telegram.org/api/links as I've done a couple of minutes ago per @ReallyNotNothing suggestion.

Reply:
Anyway, feel free to implement the change at your convenience. For the time being, I'm merging the PR since the main updates are reflected in your translation.